examples/graphql-spring-boot/src/main/kotlin/org/neo4j/graphql/examples/graphqlspringboot/config/GraphQLConfiguration.kt

```diff
@@ -0,0 +1,118 @@
+package org.neo4j.graphql.examples.graphqlspringboot.config
+
+import com.expediagroup.graphql.SchemaGeneratorConfig
```

Comment:
is that a common library used? and recommended by the graphql-java folks?

Reply:
This lib has many stars [(821)](https://github.com/ExpediaGroup/graphql-kotlin) and simplifies the generation of custom resolver methods.

With the [lib "recommented" by the graphql-java folks](https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/), the wiring of the custom resolvers would have to be done by hand, which can become time-consuming and annoying.

How would you determine if the library is recommended?